### ISSUE TITLE
VBET 0.8.9: smaller VBs mapping within the VB of longer levelpaths

### ISSUE THREAD

**Issue by shelbysawyer** [Riverscapes/riverscapes-tools#721]

**Comment by jtgilbert** [comment]
This looks like likely a calibration issue. I think that the side channel level path is being included because the mainstem VB isn't as expansive as it maybe should be.

**Comment by shelbysawyer** [comment]
Yeah, it's probably partially a calibration problem, but I'm guessing that even with calibration, this is still going to happen in some places. The problem with these little islands is not only their existence (confusing for users), but their effect on CLs and segmentation (previously documented #700). Take a look at what these islands are doing to the CL and segmentation:

![dgos](https://user-images.githubusercontent.com/73311751/225384695-bfd73344-9973-468a-96e9-741f7364af19.JPG)

1. Could we hard code the removal of islands with area values under a certain threshold value? This is something @joewheaton and I chatted about when this problem was originally raised in #700. 

2. These islands are roughly in the middle of the mainstem's valley bottom, and probably due to calibration, were excluded from the mainstem valley bottom-- but they still fall completely within that valley bottom. It doesn't make sense to me that areas such as these should be able to be run in subsequent levelpath HAND generation. If we can hard code in the removal of islands by an area value threshold, could we at least prevent the running of HAND in areas that are completely contained by an already-generated valley bottom?

3. If we can't do either of those things, could we just remove the vectors that exist completely inside a VB polygon, or would this cause issues for VBET/GNAT by ignoring true islands? @lauren-herbine tagging you because this problem is very relevant to GNAT.

**Comment by jtgilbert** [comment]
@shelbysawyer I believe there is already a part of the code that removes holes, so it might be just a question of playing with the parameters its using (its probably only filling very small ones right now).

**Comment by lauren-herbine** [comment]
Hmmmm... if we went with option 3, it would mean that true islands are removed, and in some select cases that would affect metrics such as confinement...
This is issue is definitely relevant to GNAT, thank you for documenting, @shelbysawyer 

Those holes are very odd- there are many ones far interior to the Cheat's mapped VB, so to my (less knowledgeable) eye, it does look like the Cheat VB is expansive enough to take care of them. And then HAND is being run for the shorter levelpath within the island within valley bottom of the larger levelpath, which shouldn't be happening
![image](https://user-images.githubusercontent.com/73319684/225404082-3e0c0b82-d960-42f9-89e5-24c5c270639a.png)

**Comment by lauren-herbine** [comment]
Another interesting place to look at this ([1019001003](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/855ba286-5d22-49db-8160-9d221d2fa97e)). Here it may simply be a calibration issue., but the extent of the valley bottom in other places gives me pause when jumping to that conclusion. The side trib is coded as intermittent, so it definitely should be included in our network, but I don't know if it should be getting its own VB. To me, it is wholly within the Kiowa creek VB.

![image](https://user-images.githubusercontent.com/73319684/225414652-870530b6-7eaa-4d1b-99a6-4252a19acf58.png)

Just wanted another example outside the Cheat!

**Comment by joewheaton** [comment]
Thank you @shelbysawyer and @lauren-herbine for digging into this one. My take is that 0.9.0 fixes this issue adequately. I'm going to suggest we close this unless @jtgilbert sees something else we should do.
